.env.example

```diff
@@ -102,4 +102,9 @@ DACO_ENCRYPTION_KEY=
 #############
 # Testing
 #############
-ADMIN_PAUSE=false
\ No newline at end of file
+ADMIN_PAUSE=false
+
+#############
+# Feature flags
+#############
+RENEWAL_FEATURE_ENABLED=false
```

Comment:
Good practice is to have a consistent feature flag naming scheme like `FEATURE_RENEWAL_ENABLED`.

Reply:
aha i was wondering if there was a convention - will change!

Reply:
Don't think there is a general convention, but throughout other argo apps this is how they're written so at least we're internally consistent.